src/thorin/transform/importer.h

```diff
@@ -11,6 +11,9 @@ class Importer {
     Importer(World& src)
         : world_(src)
     {
+        type_old2new_.rehash(world_.types().capacity());
```

Comment:
See also: https://github.com/AnyDSL/thorin/blob/control/src/thorin/transform/rewrite.cpp#L6

The control branch has went further into factoring out an abstract `Rewriter` class with shared logic